snomed/com.b2international.snowowl.snomed.reasoner/src/com/b2international/snowowl/snomed/reasoner/SnomedReasonerPlugin.java

```diff
@@ -31,7 +31,12 @@
 import com.b2international.snowowl.snomed.datastore.config.SnomedCoreConfiguration;
 import com.b2international.snowowl.snomed.datastore.index.constraint.SnomedConstraintDocument;
 import com.b2international.snowowl.snomed.reasoner.classification.ClassificationTracker;
-import com.b2international.snowowl.snomed.reasoner.index.*;
+import com.b2international.snowowl.snomed.reasoner.index.ClassificationTaskDocument;
```

Comment:
Configure your IDE automatic wildcard import setting and revert this change, please.